src/design-system/styles/template/block-areas/block-areas-annotate.scss

```diff
@@ -0,0 +1,42 @@
+@import "example-init";
+
+[data-block] {
+  position: relative;
+  outline: 2px dashed govuk-colour("grey-1");
+  padding: govuk-spacing(3);
+  padding-top: 3em;
+  margin: govuk-spacing(6) 0;
+}
+
+[data-block]::before {
+  content: 'block: ' attr(data-block);
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: .25em .5em;
+  background: white;
+  border-bottom: 2px dashed govuk-colour("grey-1");
+  border-right: 2px dashed govuk-colour("grey-1");
+  font-family: $govuk-font-family;
+}
+
+[data-block] > .govuk-body:last-child {
+  margin-bottom: 0;
+}
+
+[data-block] > .govuk-body {
+  line-height: 1.5;
+}
+
+.block-break {
```

Comment:
This class name could be more "BEMmy" if `data-block` were targeted with class names.

Reply:
The classes now fit BEM